libs/wingc/src/ast.rs

```diff
@@ -164,6 +164,7 @@ pub struct FunctionDefinition {
 	pub signature: FunctionSignature,
 	#[derivative(Debug = "ignore")]
 	pub captures: RefCell<Option<Captures>>,
+	pub _static: bool,
```

Comment:
Why does this have an underscore prefix?

If it is because `static` is a reserved word, then maybe call this `is_static`?

Reply:
yes, i'll fix